addons/l10n_ch/models/account_invoice.py

```diff
@@ -151,7 +150,6 @@ def _compute_l10n_ch_isr_valid(self):
             record.l10n_ch_isr_valid = record.type == 'out_invoice' and\
                 record.name and \
                 record.l10n_ch_isr_subscription and \
-                record.invoice_partner_bank_id.l10n_ch_postal and \
```

Comment:
Comment:

This line has to be removed because all payment slips don't need l10n_ch_postal (which in this case is used as a ISR-B Customer ID). See https://github.com/odoo/odoo/pull/51637 for proper implementation of ISR-B

To recap:

Case 1: ISR
requirement: l10n_ch_isr_subscription_chf

Case 2: ISR-B
requirement: l10n_ch_isr_subscription_chf + l10n_ch_isrb_id_number (introduced in a separate module)